### PR TITLE
GEOMESA-2142 Fixing --index flag in CLI export/explain

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala [hunk 1]
```diff
@@ -169,38 +166,18 @@ trait ConverterConfigParam {
...
-        // just index name
-        indices.find(_.name.equalsIgnoreCase(name))
-      }
-      matched.getOrElse {
-        throw new ParameterException(s"Specified index ' $index' not found. " +
-        s"Available indices are: ${indices.map(_.identifier).mkString(", ")}")
-      }
-    }
-  }
+  var index: String = _
```
Comment (jahhulbert-ccri) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala]: i'm about to do testing...but wondering what happens now when you pass in an index that doesn't exist?

Comment (jahhulbert-ccri) [reply geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala]: so it doesn't throw an exception if the index does not exist...perhaps it should do that?

Comment (elahrvivaz) [reply geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala]: yeah, in the log you get:
```
2018-01-17 22:59:47,421 ERROR [org.locationtech.geomesa.accumulo.index.AccumuloQueryPlanner] Ignoring invalid strategy name: foo. Valid values are z3, z3:5, z2, z2:4, records, records:2
```
should we just throw an exception there? that would affect other queries too, but arguably that would be a good thing

Comment (jahhulbert-ccri) [reply geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala]: yeah i was thinking throw an exception since it's an input error.

Comment (elahrvivaz) [reply geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala]: changed it to throw an exception

Comment (jahhulbert-ccri) [reply geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala]: yay!

**Comment by jahhulbert-ccri** [comment]
verified it throws exception...

**Pull request merged**